frontend/test/metabase/scenarios/admin/people/people.cy.spec.js

```diff
@@ -358,6 +356,78 @@ describeEE("scenarios > admin > people", () => {
     cy.findByText("Question").should("not.exist");
     cy.findByText("Dashboard").should("not.exist");
   });
+
+  describe("email configured", () => {
+    beforeEach(() => {
+      // Setup email server, since we show different modal message when email isn't configured
+      setupSMTP();
+
+      // Setup Google authentication
+      cy.request("PUT", "/api/setting", {
+        "google-auth-client-id": "fake-id.apps.googleusercontent.com",
+        "google-auth-auto-create-accounts-domain": "metabase.com",
+      });
+    });
+
+    it("invite member when SSO is not configured metabase#23630", () => {
```

Comment:
Ok, I actually have done this in a number of places 😓. I'll make sure we won't do that in the future.

I thought this would be better for referencing the reasons behind code change.